### ISSUE TITLE
Notify admin on user password change

### ISSUE THREAD

**Issue by dmidmi84** [RocketChat/Rocket.Chat#2995]
Hi!
It would be great that admin receives an e-mail or notification from rocket.cat that someone change his password.

P.S.: I can see it in Logs, but it would be great to receive it automatically.

TY!

Your Rocket.Chat version: (0.26.0)

**Comment by geekgonecrazy** [comment]
I would for sure want this to be toggle-able.   Also perhaps this would belong with an audit log view?

**Comment by AdrianoCahete** [comment]
It's a entry for audit log, not email. If you have a huge user database, sendmail will have a lot of work (and admin email too).

**Comment by geekgonecrazy** [comment]
@AdrianoCahete that's kinda what I was thinking.  An Audit log I think would be something enterprise types will for sure want.

**Comment by jakelacey2012** [comment]
Hi guys I would love to contribute to Rocket.Chat this seems like a good entry point...

I'm thinking an email for every time a password gets changed would be a bit to much, I as a user/admin would rather have an email with a summary of how many users changed their passwords with a frequency.

what do you think?

**Comment by geekgonecrazy** [comment]
@jakelacey2012 I tend to agree.  I think it would make more sense to make such activities part of an audit log.  Then do a digest with info from the audit log.

**Comment by jakelacey2012** [comment]
@geekgonecrazy Cool! I'll get onto it :+1: I'm not new to meteor but could you point me in the direction of what packages I would need to look at the implement this?

**Comment by lindoelio** [comment]
Hi Guys!

I've working on it. I spent a lot of time thinking about this feature as commented on this thread, and then I've resolved to suggest the implementation that I'm sharing on PR #7591 (in progress).

In this PR I've included a global settings for store the preference of frequency that the admins will to receive email with password change history, that can be: nothing/disabled, immediately (one email at each change action), daily (ever starting at 0:00 am - one email with summary of change actions of the last day) or weekly (ever starting at 0:00 am on first day of each week - on email with summary or change actions of the last week). All users in role "admin" will receive the password change history, as global configured.

Let's! Give-me your feedback about this :-)

**Comment by abhishek71994** [comment]
Is this feature yet to be implemented? Anyone working on it? I'd like to start.

**Comment by ashutoshk133** [comment]
I would like to Work on it
